### ISSUE TITLE
Update breadcrumbs on Discharge Upgrade form to align with new guidance

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#18861]
## Description
To align with the new/formalized guidance on Breadcrumbs we need to updated the text of the last crumb to match the H1 of the intro page.

All crumbs within the Discharge upgrade flow should be:
`VA.gov home > How to apply for a Discharge Upgrade`

**QUESTION! Are the capitals in the H1 correct? We typically use sentence case but this is using title case, was this reviewed by CAIA?**

### Additional info
Functionally the last crumb should also be going to the intro page but the v3 component does not allow for this so for now this will still be an anchor link to the current page content. [DST ticket for functionality update
](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3114)

## Acceptance Criteria
- [ ] First crumb on all screens of the form flow says "VA.gov home"
- [ ] Second/last crumb on all screens of the form flow says "How to apply for a Discharge Upgrade"
- [ ] Requires accessibility review

**Comment by davidmpickett** [comment]
The capitalization of "How to Apply for a Discharge Upgrade" came from the content document. It does seems like an oversight though. Discharge upgrade is sentence case on the [page that leads to the wizard](https://www.va.gov/resources/request-a-discharge-upgrade-or-correction/).

**Comment by FranECross** [comment]
@FranECross Follow up on capitalization question.

**Comment by chriskim2311** [comment]
@FranECross Did we get an answer from CAIA for the capitalization? Thank you!

**Comment by FranECross** [comment]
@chriskim2311 I can't find the convo, but it shouldn't be title case; should be sentence case. Thanks for pinging me! I meant to come back and let you know.

**Comment by laflannery** [comment]
Confirmed in Prod on current wizard that breadcrumbs have been updated. Also confirmed in staging on new form that breadcrumbs are correct
